pom.xml

```diff
@@ -5,11 +5,12 @@
     <version>1.2.1</version>
     <packaging>jar</packaging>
     <name>Amazon DynamoDB Lock Client</name>
-    <url>https://github.com/awslabs/dynamodb-lock-client</url>
+    <url>https://github.com/awslabs/amazon-dynamodb-lock-client</url>
```

Comment:
Why do we need this change? Was the repo renamed by someone already?

Reply:
Not a needed change, but this is just what the new URL is when looking at the new lockclient library url. Though both URLs lead to the same repo. I guess the url was changed at some point